src/styles/tokens/colors.scss

```diff
@@ -28,6 +28,7 @@
   --sb-disabled-background-color: #ecedf5;
   --sb-primary-background-hover-color: #dcdfec;
   --sb-dark-background-color: #f6f7fb;
+  --sb-brand-color: #5034ff;
```

Comment:
Nice!
Let's also create a task to replace this hard-coded value to this token in vibe docs? We can put it for the hackathon :)

Reply:
Good idea :)